Reconstruction/RecTracker/RecPlotsMakefile.mk

```diff
@@ -0,0 +1,28 @@
+###########################
+# FCC Track Reco Plots Makefile
```

Comment:
I like having the bash commands in the makefile as it's easy to keep them in sync and they're testable, but it's true that RecTracker was still missing some docs. It will have to be expanded, but I pushed a start (Reconstruction/doc/RecTracker.md)